common-app-lib/src/cmr/common_app/api/health.clj

```diff
@@ -34,7 +35,8 @@
   (when-not ok?
     (warn "Health check failed" (pr-str dependencies)))
   {:status (if ok? 200 503)
-   :headers {"Content-Type" (mt/with-utf-8 mt/json)}
+   :headers {common-routes/CONTENT_TYPE_HEADER (mt/with-utf-8 mt/json)
+             common-routes/CORS_ORIGIN_HEADER "*"}
```

Comment:
I'm not adding other CORS headers here because I think those are related to OPTIONS case. like CORS_METHODS_HEADER, CORS_MAX_AGE_HEADER.. right?

Reply:
yes.